test.py

```diff
@@ -1 +1,2 @@
 print("This is a TEST!")
+print("This is another TEST!")
```

Comment:
@cam2-code-review-bot assign "huang824" "LOGIC" "Test whether the bot can be mentioned only within the review comment"